execution-env.md

```diff
@@ -722,3 +724,1149 @@ Note: The **std430** in GLSL satisfies these rules.
+   </td>
+   <td>OpSpecConstant
+   </td>
+  </tr>
+  <tr>
+   <td>51
+   </td>
+   <td>OpSpecConstantComposite
+   </td>
+  </tr>
+  <tr>
+   <td>52
+   </td>
+   <td>OpSpecConstantOp
+   </td>
+  </tr>
+  <tr>
+   <td>54
+   </td>
+   <td>OpFunction
+   </td>
+  </tr>
+  <tr>
+   <td>55
+   </td>
+   <td>OpFunctionParameter
+   </td>
+  </tr>
+  <tr>
+   <td>56
+   </td>
+   <td>OpFunctionEnd
+   </td>
+  </tr>
+  <tr>
+   <td>57
+   </td>
+   <td>OpFunctionCall
+   </td>
+  </tr>
+  <tr>
+   <td>59
+   </td>
+   <td>OpVariable
+   </td>
+  </tr>
+  <tr>
+   <td>60
+   </td>
+   <td>OpImageTexelPointer
+   </td>
+  </tr>
+  <tr>
+   <td>61
+   </td>
+   <td>OpLoad
+   </td>
+  </tr>
+  <tr>
+   <td>62
+   </td>
+   <td>OpStore
+   </td>
+  </tr>
+  <tr>
+   <td>63
+   </td>
+   <td>OpCopyMemory
+   </td>
+  </tr>
+  <tr>
+   <td>65
+   </td>
+   <td>OpAccessChain
+   </td>
+  </tr>
+  <tr>
+   <td>66
+   </td>
+   <td>OpInBoundsAccessChain
+   </td>
+  </tr>
+  <tr>
+   <td>68
+   </td>
+   <td>OpArrayLength
+   </td>
+  </tr>
+  <tr>
+   <td>71
+   </td>
+   <td>OpDecorate
+   </td>
+  </tr>
+  <tr>
+   <td>72
+   </td>
+   <td>OpMemberDecorate
+   </td>
+  </tr>
+  <tr>
+   <td>77
+   </td>
+   <td>OpVectorExtractDynamic
+   </td>
+  </tr>
+  <tr>
+   <td>78
+   </td>
+   <td>OpVectorInsertDynamic
+   </td>
+  </tr>
+  <tr>
+   <td>79
+   </td>
+   <td>OpVectorShuffle
+   </td>
+  </tr>
+  <tr>
+   <td>80
+   </td>
+   <td>OpCompositeConstruct
+   </td>
+  </tr>
+  <tr>
+   <td>81
+   </td>
+   <td>OpCompositeExtract
+   </td>
+  </tr>
+  <tr>
+   <td>82
+   </td>
+   <td>OpCompositeInsert
+   </td>
+  </tr>
+  <tr>
+   <td>83
+   </td>
+   <td>OpCopyObject
+   </td>
+  </tr>
+  <tr>
+   <td>84
+   </td>
+   <td>OpTranspose
+   </td>
+  </tr>
+  <tr>
+   <td>86
+   </td>
+   <td>OpSampledImage
+   </td>
+  </tr>
+  <tr>
+   <td>87
+   </td>
+   <td>OpImageSampleImplicitLod
+   </td>
+  </tr>
+  <tr>
+   <td>88
+   </td>
+   <td>OpImageSampleExplicitLod
+   </td>
+  </tr>
+  <tr>
+   <td>89
+   </td>
+   <td>OpImageSampleDrefImplicitLod
+   </td>
+  </tr>
+  <tr>
+   <td>90
+   </td>
+   <td>OpImageSampleDrefExplicitLod
+   </td>
+  </tr>
+  <tr>
+   <td>91
+   </td>
+   <td>OpImageSampleProjImplicitLod
+   </td>
+  </tr>
+  <tr>
+   <td>92
+   </td>
+   <td>OpImageSampleProjExplicitLod
+   </td>
+  </tr>
+  <tr>
+   <td>93
+   </td>
+   <td>OpImageSampleProjDrefImplicitLod
+   </td>
+  </tr>
+  <tr>
+   <td>94
+   </td>
+   <td>OpImageSampleProjDrefExplicitLod
+   </td>
+  </tr>
+  <tr>
+   <td>95
+   </td>
+   <td>OpImageFetch
+   </td>
+  </tr>
+  <tr>
+   <td>96
+   </td>
+   <td>OpImageGather
+   </td>
+  </tr>
+  <tr>
+   <td>97
+   </td>
+   <td>OpImageDrefGather
+   </td>
+  </tr>
+  <tr>
+   <td>98
+   </td>
+   <td>OpImageRead
+   </td>
+  </tr>
+  <tr>
+   <td>99
+   </td>
+   <td>OpImageWrite
+   </td>
+  </tr>
+  <tr>
+   <td>100
+   </td>
+   <td>OpImage
+   </td>
+  </tr>
+  <tr>
+   <td>105
+   </td>
+   <td>OpImageQueryLod
```

Comment:
ImageQuery capability is allowed, and that also permits the following instructions:

- OpImageQuerySizeLod
- OpImageQuerySize
- OpImageQueryLevels
- OpImageQuerySamples

Reply:
Done. I have manually merged them here and will have to look at handling this correctly in my spreadsheet.